src/ru/skypro/Main.java

```diff
@@ -4,5 +4,82 @@
 
     public static void main(String[] args) {
 	// write your code here
+
+        //Task1
+
+        int clientOS = 0;
+        if (clientOS==0) {
+            System.out.println("Установите версию приложения для iOS по ссылке");
+        }
+        if (clientOS==1) {
+            System.out.println("Установите версию приложения для Android по ссылке");
+        }
+
+        // Task2;
+
+        int clientDeviceYear = 2015;
+        if (clientOS==0 && clientDeviceYear < 2015) {
+            System.out.println("Установите облегченную версию приложения для iOS по ссылке");
+        } else if (clientOS==0 && clientDeviceYear >=2015) {
+            System.out.println("Установите версию приложения для iOS по ссылке");
+        }
+        if(clientOS==1 && clientDeviceYear < 2015) {
+            System.out.println("Установите облегченную версию приложения для Android по ссылке");
+        } else if(clientOS==1 && clientDeviceYear >= 2015) {
+            System.out.println("Установите версию приложения для Android по ссылке");
+        }
+
+        //Task3
+
+        int year = 2021;
+        int a = year % 4;
+        int b = year % 400;
+        int c = year % 100;
+
+        if (c != 0 && a == 0 || c !=0 && b == 0) {
+            System.out.println(year + " год является високосным.");
+        }
+        else  {
+            System.out.println(year + " год не является високосным.");
+        }
+
+        //Task4
+
+        int deliveryDistance= 95;
+        if (deliveryDistance <  20) {
+            System.out.println("Потребуется дней: " + 1);
+        } else if (20 <= deliveryDistance && deliveryDistance < 60) {
+            System.out.println("Потребуется дней: " + 2);
+        } else if (60 <= deliveryDistance && deliveryDistance <= 100) {
```

Comment:
```suggestion
        } else if (deliveryDistance < 60) {
            System.out.println("Потребуется дней: " + 2);
        } else if (deliveryDistance <= 100) {
```
поскольку мы используем единый оператор, то левые условия автоматически выполняются из-за невыполнения условия выше, можно упростить